numpy/add_newdocs.py

```diff
@@ -7756,3 +7758,36 @@ def luf(lamdaexpr, *args, **kwargs):
 
 add_newdoc('numpy.core.numerictypes', 'object_',
     """Any Python object.  Character code: 'O'.""")
+
```

Comment:
Directly sets the `__doc__` attributes of the target at run time. No generation happens here.